hexa/databases/api.py

```diff
@@ -0,0 +1,71 @@
+import psycopg2
+from django.conf import settings
+from psycopg2 import sql
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+
+
+def get_db_server_credentials():
+    return {
+        "role": settings.WORKSPACE_DATABASE_ROLE,
+        "password": settings.WORKSPACE_DATABASE_PASSWORD,
+        "host": settings.WORKSPACE_DATABASE_HOST,
+        "port": settings.WORKSPACE_DATABASE_PORT,
+    }
+
+
+def create_database(name, pwd):
+    """
+    Create a database and role associated to it
+    Args :
+    name - database name (it will be used also for the role name)
+    pwd  - password used by the created role to connect to the db
+    """
+    credentials = get_db_server_credentials()
+
+    role = credentials["role"]
+    password = credentials["password"]
+    host = credentials["host"]
+    port = credentials["port"]
+
+    url = f"postgresql://{role}:{password}@{host}:{port}"
+
+    db_name = ""
```

Comment:
It's weird that you changes the name of the database you receive. How do the caller know that you maybe adapted the name of the database ? IMHO it should be a helper and it should only be validated here (and raises an error if it's not correct)

Reply:
Yes, you right,.

Reply:
Also, now that we use `sql.Identifier`, no need to be super paranoid about the dbname - it should prevent SQL injection. Using the workspace slug should be fine.